packages/livekit-rtc/generate_proto.sh

```diff
@@ -5,7 +5,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # This script requires protobuf-compiler and https://www.npmjs.com/package/@bufbuild/protoc-gen-es
-# `brew install protobuf-c && npm install -g @bufbuild/protoc-gen-es@1.10.0`
+# `brew install protobuf-c && npm install -g @bufbuild/protoc-gen-es@2.2.0`
```

Comment:
@lukasIO can you just make this change in your fix PR https://github.com/livekit/node-sdks/pull/310